string agent_id
geometry_msgs/Pose2D goal_pose
---
geometry_msgs/Pose2D[] path
